components/dashboard/tailwind.config.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
     important: true,
     darkMode: "class",
     theme: {
+        maxWidth: {
+            // TODO(andreafalzetti): remove custom ide-modal class once we implement https://github.com/gitpod-io/gitpod/issues/13116
+            'ide-modal': '51.5rem',
```

Comment:
❗ **issue**: Not sure if this is the best line to comment about this, but the new modal width for the editor selection affects most modals inside the dashboard.

| Open workspace | New variable | Integrations | ... |
|-|-|-|-|
| <img width="1440" alt="Screenshot 2022-11-10 at 4 39 09 PM" src="https://user-images.githubusercontent.com/120486/201120770-8d2b3eff-7ab0-4fa9-ac68-6007c584313f.png"> | <img width="1440" alt="Screenshot 2022-11-10 at 4 39 21 PM" src="https://user-images.githubusercontent.com/120486/201120782-bd534570-65ff-4629-8728-bdd5bfc4b836.png"> | <img width="1440" alt="Screenshot 2022-11-10 at 4 39 28 PM" src="https://user-images.githubusercontent.com/120486/201120948-68aeb70b-cd87-4cd3-8aa2-ff7df0994f04.png"> | ... |

Reply:
FYI, pr still in draft but I've pushed a fix for this 🙏

Reply:
Thanks, @andreafalzetti! LGTM now! Let me know if this could benefit from an extra pair of eyes to go through UX changes when this gets ready for review. 🏀